builder/vmware/iso/builder.go

```diff
@@ -259,6 +259,7 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 			HTTPPortMax: b.config.HTTPPortMax,
 		},
 		&vmwcommon.StepConfigureVNC{
+			Skip:               b.config.BootCommand == nil,
```

Comment:
Could you make `b.config.BootCommand == nil` a variable defined at the top of the function?